src/code/z_actor.c

```diff
@@ -776,25 +777,25 @@ void Actor_Kill(Actor* actor) {
 }
 
 void Actor_InitPosRot(Actor* actor) {
-    actor->posRot = actor->initPosRot;
+    actor->world = actor->home;
 }
 
 void Actor_SetHeight(Actor* actor, f32 offset) {
```

Comment:
This function should probably also be renamed based on the name we go with for posRot2